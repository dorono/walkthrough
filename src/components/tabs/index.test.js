import React from 'react';
import {shallow} from 'enzyme';
import Tabs from './index';

describe('Tabs', () => {
    it('should render without errors', () => {
        shallow(<Tabs items={[]} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Tabs items={[]} />)).toMatchSnapshot();
    });
});
