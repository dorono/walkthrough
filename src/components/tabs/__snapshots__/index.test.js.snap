// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs should match snapshot 1`] = `
<div
  className="root"
/>
`;
